### DISCUSSION TITLE
Release for newest gnome version.

### DISCUSSION THREAD

**Discussion by micahosborne** [GnomeSnapExtensions/gSnap#71] [General]
Hey @meronz,  any plan to get this released?  I would do it but i dont have the account to publish anymore ;)

**Comment by meronz** [comment]
I'll do it on the weekend since I am currently away and I don't have access to a computer right now!

**Reply by micahosborne** [reply]
Okay good deal! Thank you!

**Comment by meronz** [comment]
I've pushed the release [releases/14](https://extensions.gnome.org/review/43461) to extensions.gnome.org. We are waiting for the review :)

**Comment by micahosborne** [comment]
Thank you sir!
